--- contrib/poco/Net/src/HTTPBufferAllocator.cpp	2026-04-20 15:36:08.517657297 +0000
+++ contrib/poco-cmake/patches/HTTPBufferAllocator.cpp.new	2026-04-20 16:41:39.868284083 +0000
@@ -15,21 +15,19 @@
 #include "Poco/Net/HTTPBufferAllocator.h"
 
 
-using Poco::MemoryPool;
 
 
 namespace Poco {
 namespace Net {
 
 
-MemoryPool HTTPBufferAllocator::_pool(HTTPBufferAllocator::BUFFER_SIZE, 16);
 
 
 char* HTTPBufferAllocator::allocate(std::streamsize size)
 {
 	poco_assert_dbg (size == BUFFER_SIZE);
 
-	return reinterpret_cast<char*>(_pool.get());
+	return new char[static_cast<std::size_t>(size)];
 }
 
 
@@ -37,7 +35,7 @@
 {
 	poco_assert_dbg (size == BUFFER_SIZE);
 
-	_pool.release(ptr);
+	delete[] ptr;
 }
 
 
